src/clarity-angular/popover/popover.ts

```diff
@@ -249,11 +242,11 @@ export class Popover {
         }
 
         this.element.style.transform = `translateX(${leftDiff}px) translateY(${topDiff}px)`;
+        return this._scroll.asObservable();
     }
 
     public destroy() {
-        this.element.style.transform = "none";
-        this.resumeScrolling();
```

Comment:
Is there a reason why you removed the `this.element.style.transform` cleanup? The destroy here is not guaranteed to be called only when the element is destroyed, so we might leave an element out there with unwanted styles.